portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,62 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Gochi+Hand&family=Manrope&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <header id="intro">
+            <h3>Hi I'm</h3>
+            <h2 id="name"> Rebecca Sun</h2>
+        </header>
+
+        <h4 id="categories">-ABOUT-</h4>
+        <div id="about">
+            <img id="headshot" src="images/me.jpg"></img>
+            <p>Currently, I am a <a href="https://buildyourfuture.withgoogle.com/programs/step/">STEP</a> intern at Google. 
+            I am a rising junior at Vanderbilt University majoring in computer science with a minor in engineering management. 
+            This summer, I'm working out of Kansas City, Kansas where I'm from.</p>
+        </div>
+        
+        <h4 id="categories">-EXPERIENCE-</h4>
+        <ul id="info">
+            <li>Summer 2019: Software Engineer Intern at Atonix Digital</li>
+            <li>Summer 2019: Google CSSI-Coursera TA</li>
+            <li>December 2019-Present: Vanderbilt Women in Computing (ACM-W Chapter) President</li>
+        </ul>
+        <h4 id="categories">-SKILLS & TECHNICAL LANGUAGES-</h4>
+        <ul id="info"> 
+            <li>C++</li>
+            <li>Python</li>
+            <li>Java</li>
+            <li>HTML, CSS, JavaScript</li>
+        </ul>
+        <h4 id="categories">-PROJECTS-</h4>
+        <ul id="info"> 
+            <li>VandyHacks submission: TA Office Hours Scheduler Chrome Extension</li>
+            <li>MIT Women's Technology Program Final Project: Tetris program written in Python</li>
+        </ul>
+        <h4 id="categories">-INTERESTS & HOBBIES-</h4>
+        <ul id="info"> 
+            <li>Hiking</li>
+            <li>Skiing</li>
+            <li>Cooking and baking</li>
+            <li>Fun fact: I fenced competitively for 7 years</li>
+        </ul>
     </div>
+
+    <div id="profiles">
```

Comment:
Nothing to change here =)  I just wanted to note that this is a good example of when to use both an id and a plural name.  Using an id is good because you only have one section on your page dedicated to profiles, so the id uniquely identifies this spot.  And the plural is good because this section is inherently something containing a list multiple of... profiles, so "profiles" seems an ideal name!

Reply:
makes sense!